cpp/tests/groupby/group_mean_test.cpp

```diff
@@ -137,5 +138,27 @@ TYPED_TEST(groupby_mean_test, null_keys_and_values)
 }
 // clang-format on
 
+struct groupby_dictionary_mean_test : public cudf::test::BaseFixture {
+};
+
+TEST_F(groupby_dictionary_mean_test, basic)
```

Comment:
This test seems like a placeholder until 10.2 is removed, or the bug is fixed.
if this test needs to be disabled, prefix DISABLED_ to the name.

Reply:
Cool. Thanks for this.